pyvela/pyvela/pyvela_script.py

```diff
@@ -134,22 +141,28 @@ def validate_input(args):
 
 
 def prepare_outdir(args):
-    if args.force_rewrite and os.path.isdir(args.outdir):
+    if args.force_rewrite and os.path.isdir(args.outdir) and not args.restart:
         shutil.rmtree(args.outdir)
 
-    os.mkdir(args.outdir)
+    if not args.restart and not os.path.isdir(args.outdir):
+        os.mkdir(args.outdir)
 
-    shutil.copy(args.par_file, args.outdir)
+    if not os.path.exists(f"{args.outdir}/args.par_file"):
+        shutil.copy(args.par_file, args.outdir)
```

Comment:
This will not work if `args.par_file` is an absolute path. Please use `os.path.basename(args.par_file)` in `shutil.copy()`.

Reply:
I think the abs path needs to go in the check for existence - that's what I did.